src/server.rs

```diff
@@ -162,8 +162,13 @@ mod test {
   #[test]
   fn it_makes_a_server() {
     let server = Server::new(|_| Response::BeginScenario);
-    let (handle, stop_tx) = server.start(Some("0.0.0.0:1234"));
-    let _ = TcpStream::connect("0.0.0.0:1234").unwrap();
+    let addr = if cfg!(target_os = "windows") {
+      "127.0.0.1:1234"
+    } else {
+      "0.0.0.0:1234"
+    };
```

Comment:
Ah can you get these test ones too?

If you dont get around to it, i'll make the fix and merge up.